packages/components/src/ButtonGroup/__stories__/ButtonGroup.stories.js

```diff
@@ -1,4 +1,4 @@
-import { FiBold, FiItalic, FiUnderline } from '@wp-g2/icons';
+import { formatBold, formatItalic, formatUnderline } from '@wordpress/icons';
```

Comment:
`formatUnderline` exists as of 2 weeks ago and should be available in the next release. Though currently it renders nothing.

Reply:
That's alright :). It's not too important that they work for the stories.